pattern-filter/apps/pattern-filter/src/main/java/com/redhat/training/transform/Application.java

```diff
@@ -0,0 +1,203 @@
+package com.redhat.training.transform;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+import com.redhat.training.model.OrderItem;
+import com.redhat.training.model.CatalogItem;
+import com.redhat.training.model.Order;
+import com.redhat.training.model.Customer;
+
+//import org.apache.activemq.ActiveMQConnectionFactory;
+import org.apache.activemq.artemis.jms.client.ActiveMQConnectionFactory;
+
+import javax.jms.Connection;
+import javax.jms.DeliveryMode;
+import javax.jms.Destination;
+import javax.jms.ExceptionListener;
+import javax.jms.JMSException;
+import javax.jms.Message;
+import javax.jms.MessageConsumer;
+import javax.jms.MessageProducer;
+import javax.jms.Session;
+import javax.jms.TextMessage;
+import javax.jms.ObjectMessage;
+
+import java.math.BigDecimal;
+import java.lang.Integer;
+
+@SpringBootApplication
+public class Application {
+
+	public static void main(String[] args) {
+		initializeAMQ();
+		SpringApplication.run(Application.class, args);
+	}
+
+	public static void initializeAMQ() {
+		System.out.println("-------------------- Initializing AMQ ----------------------");
+		thread(
+			new OrderProducer(
+				new Integer(1),
+				new BigDecimal("0.01"),
+				true,
+				new Customer(
+					"Frank",
+					"Smith",
+					"fsmith",
+					"password",
+					"fsmith@email.com",
+					false
+				)
+			),
+			false
+		);
+		thread(
+			new OrderProducer(
+				new Integer(2),
+				new BigDecimal("0.01"),
+				true,
+				new Customer(
+					"John",
+					"Smith",
+					"jsmith",
+					"password",
+					"jsmith@email.com",
+					false
+				)
+			),
+			false
+		);
+		thread(
+			new OrderProducer(
+				new Integer(3),
+				new BigDecimal("0.02"),
+				true,
+				new Customer(
+					"Jerry",
+					"Jones",
+					"jjones",
+					"password",
+					"jjones@email.com",
+					false
+				)
+			),
+			false
+		);
+		thread(
+			new OrderProducer(
+				new Integer(4),
+				new BigDecimal("0.015"),
+				false,
+				new Customer(
+					"Jess",
+					"Thompson",
+					"jthompson",
+					"password",
+					"jthompson@email.com",
+					false
+				)
+			),
+			true
+		);
+		thread(
+			new OrderProducer(
+				new Integer(5),
+				new BigDecimal("0.012"),
+				false,
+				new Customer(
+					"Tony",
+					"Anderson",
+					"tanderson",
+					"password",
+					"tanderson@email.com",
+					false
+				)
+			),
+			true
+		);
+
+	}
+
+	public static OrderItem getOrderItem(int id, int qty, float price, CatalogItem item) {
+		OrderItem o = new OrderItem();
+		o.setId(new Integer(id));
+		o.setQuantity(new Integer(qty));
+		o.setExtPrice(new BigDecimal(price));
+		o.setItem(item);
+		return o;
```

Comment:
For clarity, specially with students reading our code, I think that we should always have an empty line before the returns.